libs/ui-toolkit/modals/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@gorilainvest/ui-toolkit-modals",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^7.2.0",
+    "@angular/core": "^7.2.0"
+  },
+  "scripts": {
```

Comment:
Libraries doesn't need to have npm scripts. All angular cli commands you can run it in repository root